boxes/boxes/react/tests/node.test.ts

```diff
@@ -21,9 +21,7 @@ describe('BoxReact Contract Tests', () => {
 
   test('Can set a number', async () => {
     logger(`${await wallet.getRegisteredAccounts()}`);
-    const callTxReceipt = await contract.methods.setNumber(numberToSet, wallet.getCompleteAddress()).send().wait();
-
-    expect(callTxReceipt.status).toBe(TxStatus.MINED);
```

Comment:
Forgot to nuke this one in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/5271) so I sneaked that in here.